archive-exhibition.php

```diff
@@ -0,0 +1,134 @@
+<?php
+get_header();
+?>
+
+<main id="main-content">
+  <section id="posts">
+    <div class="container">
+
+<?php
+$now = time();
+
+$current_args = array(
```

Comment:
are these args the same as frontpage logics? Perhaps worth making a function that returns either the WP_Queries or maybe these arg arrays?